### PR TITLE
[DON'T SQUASH] Some improvements and fixes [WIP]

### LINKED ISSUE #137: Teleport sounds plays when teleport fails
If you click on “go to home” but don't have a home pos set yet, you don't teleport, but you still hear the teleport sound.

Version: 04b1cb9

### PR DESCRIPTION
**Pull request by MoNTE48**
!!! WIP !!!
1) Cleanup
2) Code-style improvements
3) item_names: do not handle MultiCraft client
4) Fix Teleport sounds when teleport fails (https://github.com/minetest-mods/unified_inventory/issues/137)
5) ...

Fixes #137.

### CODE REVIEW

File: bags.lua [hunk 1]
```diff
@@ -229,36 +229,36 @@ end)
...
 })
 
 -- register bag crafts
 if minetest.get_modpath("farming") ~= nil then
 	minetest.register_craft({
 		output = "unified_inventory:bag_small",
 		recipe = {
-			{"",           "farming:string", ""},
-			{"group:wool", "group:wool",     "group:wool"},
-			{"group:wool", "group:wool",     "group:wool"},
```
Comment (SmallJoker) [inline bags.lua:254]: FYI: This spacing is intended to visually line up the ingredients like a grid.

Comment (MoNTE48) [reply bags.lua:254]: Oh, ok. It was not very aligned in my editor. I will revert this change.

File: item_names.lua [hunk 2]
```diff
@@ -39,7 +48,16 @@ end)
...
+		local player_name = player:get_player_name()
+
+		-- Do not handle MultiCraft client
+		local info = minetest.get_player_information(player_name)
+		if info and info.version_string and info.version_string > "1.6.0"
+				and info.version_string < "2.9.9" then
+			return
+		end
+
+		local data = item_names[player_name]
```
Comment (MoNTE48) [inline item_names.lua:60]: ToDo. Better solution (maybe) is set `item_names[player_name].disable = true`.

**Comment by Panquesito7** [comment]
@MoNTE48 in the first post add `Fixes #137`.
Also, rebase is needed, please rebase.

**Comment by MoNTE48** [comment]
I hope to find some time and finish the PR later. And also move item_names commit to a separate PR.

**Comment by SmallJoker** [comment]
Sorry for neglecting this PR. In the meantime, the teleporting sounds were fixed by you ( f254269 ) and the code has seen quite some cleanups.
If there's any changes that still need to be done, feel free to open a new PR. Rebasing might be slower in this case.

**Pull request closed without merging**